contracts/HubPool.sol

```diff
@@ -47,14 +83,41 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
     );
     event WhitelistRoute(address originToken, uint256 destinationChainId, address destinationToken);
 
-    constructor(address _l1Weth, address _timerAddress) Testable(_timerAddress) {
+    event RelayerRefundRequested(
+        uint64 relayerRefundId,
+        uint256[] bundleEvaluationBlockNumbers,
+        bytes32 indexed poolRebalanceProof,
+        bytes32 indexed destinationDistributionProof,
+        address proposer
+    );
+
+    constructor(
+        uint64 _bondTokenFinalFeeMultiplier,
+        address _l1Weth,
+        address _bridgeAdmin,
+        address _bondToken,
+        address _timerAddress
+    ) Testable(_timerAddress) {
+        bondTokenFinalFeeMultiplier = _bondTokenFinalFeeMultiplier;
         l1Weth = WETH9Like(_l1Weth);
+        bridgeAdmin = BridgeAdminInterface(_bridgeAdmin);
+        bondToken = IERC20(_bondToken);
     }
 
     /*************************************************
      *                ADMIN FUNCTIONS                *
      *************************************************/
 
+    function setBondToken(address newBondToken) public onlyOwner {
+        bondToken = IERC20(newBondToken);
+        emit BondTokenSet(newBondToken);
+    }
+
+    function setBondTokenFinalFeeMultiplier(uint64 newBondTokenFinalFeeMultiplier) public onlyOwner {
```

Comment:
This is interesting. I totally get the rationale: it's easier to just set the final fees and have this move around as a result since they're kinda a valuation anchor, but do you think this is weird in that it relies on internal UMA system variables for setting its own security bounds? Like if the UMA voting system changed and spam were less of a concern, so final fees were dropped by 95%, we probably wouldn't want this value to change as a result, right?

It's also not _super_ hard to manage this since it's only one bond and one token for the entire Across v2 system.

As a side note: this also means that we need to be sure that the bond token is an approved UMA collateral currency.

Reply:
this is all true! would you prefer it to simply have one number that is set within the contract and we don't pull anything from the store? my logic was that the store is a _reasnoble_ bond size that can be scaled. we can easily just define one number in this contract that sets the bond for everything

Reply:
OK I've changed this. we now have a `bondAmount` that the owner can set. much cleaner

Reply:
agreed with this, its cleaner and in the same spirit of the UMA final fee